recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
I'm generally against `make -j` because I think `make` is not smart enough so it can screw the build process sometimes but I do not have any concrete proof.